### DISCUSSION TITLE
Installation question

### DISCUSSION THREAD

**Discussion by daveberry-dn** [houdiniproject/houdini#1072] [Q&A]
I've got this installed this afternoon.  I'm not an experienced programmer but I fixed the issues one by one!
 ran bin/rails houdini:nonprofit:create and answered the questions.  Our non-profit is in the UK, but the two letter code has to be US.  Does this work outside the US and if so what's the best course of action here?

**Comment by RamtinKosari** [comment]
Hi @daveberry-dn 

Houdini is a tool that simplifies the setup of a nonprofit organization website with the Stripe payment processing integration. While Houdini was primarily designed for nonprofits based in the United States, it is possible to use it for nonprofits outside the US as well.

When running the bin/rails houdini:nonprofit:create command, you were prompted to enter a two-letter country code. The country code is used by Houdini to set up certain configurations specific to the United States. However, this does not restrict the usage of Houdini solely to US-based nonprofits.

If your nonprofit is based in the UK and you entered the US country code during the setup process, it may affect some aspects of the default configurations provided by Houdini. For example, the default tax settings, currency, or address formatting might be tailored for the United States.

To adapt Houdini for your UK-based nonprofit, here are a few steps you can take:

Update the country-specific configurations: Review the Houdini documentation or configuration files to identify any country-specific settings that need adjustment. This may include tax settings, currency symbols, address formats, or any other relevant configurations. Make the necessary modifications to align with the requirements for UK-based nonprofits.

Customize the template: Houdini generates a basic website template for your nonprofit. You can further customize the template to reflect your UK-based nonprofit's branding, content, and any specific requirements. Modify the templates, stylesheets, and other frontend assets as needed to suit your nonprofit's needs.

Review legal and compliance requirements: Ensure that your nonprofit complies with all the legal and compliance obligations specific to the UK. This may include tax regulations, data protection laws, privacy policies, and any other relevant regulations. Make the necessary updates to ensure compliance.

Test and validate: Once you have made the above adjustments, thoroughly test the website and payment processing functionality. Ensure that the payment flows, tax calculations (if applicable), and other features are working correctly for your UK-based nonprofit. Perform test transactions and review the results to validate that everything is functioning as expected.

Remember, as you mentioned being a beginner programmer, it's essential to seek guidance or assistance from experienced developers or professionals familiar with nonprofit compliance in the UK. They can help you navigate any specific legal or technical requirements unique to your region.

Additionally, the Houdini project itself may have a community or support channels where you can seek guidance from other users or the project maintainers. Utilize these resources to ask questions, share your experiences, and get assistance with any challenges you encounter during the setup process.

Best of luck with your nonprofit website setup, and I hope your UK-based nonprofit finds success with Houdini!
